io.github.CatalystDevOrg.Catalyst/io.github.CatalystDevOrg.Catalyst.appdata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.CatalystDevOrg.Catalyst</id>
+  
+  <name>Catalyst</name>
+  <summary>A lightweight and user-focused web browser powered by Electron</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      Catalyst web browser
```

Comment:
how about a more comprehensive description?

Reply:
The description seems to of been pulled directly from the repository. I didn't put much thought into the description.

Reply:
Updated